notification/app/notification/services/guardian/ShardedNotification.scala

```diff
@@ -0,0 +1,10 @@
+package notification.services.guardian
+
+import models.Notification
+
+case class ShardRange(start: Short, end: Short)
```

Comment:
We would need to move this `ShardRange` model into `common` so it can be used by the `db` logic and the `workers`